pkg/validate/container.go

```diff
@@ -64,137 +64,137 @@ var _ = framework.KubeDescribe("Container", func() {
-			testStopContainer(rc, containerID)
-		})
-
-		It("runtime should support removing container [Conformance]", func() {
-			By("create container")
-			containerID := createDefaultContainer(rc, ic, podID, podConfig, "container-for-remove-test-")
-
-			By("test remove container")
-			removeContainer(rc, containerID)
-			containers := listContainerForID(rc, containerID)
-			Expect(containerFound(containers, containerID)).To(BeFalse(), "Container should be removed")
-		})
-
-		It("runtime should support execSync [Conformance]", func() {
-			By("create container")
-			containerID := createDefaultContainer(rc, ic, podID, podConfig, "container-for-execSync-test-")
-
-			By("start container")
-			startContainer(rc, containerID)
-
-			By("test execSync")
-			testExecSync(rc, containerID)
-		})
-	})
-
-	Context("runtime should support adding volume and device [Conformance]", func() {
-		var podID string
-		var podConfig *runtimeapi.PodSandboxConfig
-
-		BeforeEach(func() {
-			podID, podConfig = createPodSandboxForContainer(rc)
-		})
-
-		AfterEach(func() {
-			By("stop PodSandbox")
-			rc.StopPodSandbox(podID)
-			By("delete PodSandbox")
-			rc.RemovePodSandbox(podID)
-		})
-
-		It("runtime should support starting container with volume [Conformance]", func() {
-			By("create host path and flag file")
-			hostPath, flagFile := createHostPath(podID)
-
-			defer os.RemoveAll(hostPath) // clean up the TempDir
-
-			By("create container with volume")
-			containerID := createVolumeContainer(rc, ic, "container-with-volume-test-", podID, podConfig, hostPath, flagFile)
-
-			By("test start container with volume")
-			testStartContainer(rc, containerID)
-
-			By("test container exit code")
-			Eventually(getContainerStatus(rc, containerID).ExitCode).Should(Equal(int32(0)))
-		})
-	})
-
-	Context("runtime should support log", func() {
-		var podID, hostPath string
-		var podConfig *runtimeapi.PodSandboxConfig
-
-		BeforeEach(func() {
-			podID, podConfig, hostPath = createPodSandboxWithLogDirectory(rc)
-		})
-
-		AfterEach(func() {
-			By("stop PodSandbox")
-			rc.StopPodSandbox(podID)
-			By("delete PodSandbox")
-			rc.RemovePodSandbox(podID)
-			By("clean up the TempDir")
-			os.RemoveAll(hostPath)
-		})
-
-		It("runtime should support starting container with log [Conformance]", func() {
-			By("create container with log")
-			logPath, containerID := createLogContainer(rc, ic, "container-with-log-test-", podID, podConfig)
-
-			By("start container with log")
-			startContainer(rc, containerID)
-			// wait container started and check the status.
-			Eventually(verifyContainerStatus(rc, containerID, runtimeapi.ContainerState_CONTAINER_RUNNING), pollTIMEOUT).Should(BeTrue())
-
-			By("check the log context")
-			expectedLogMessage := &logMessage{
-				log:    []byte(defaultLog + "\n"),
-				stream: stdoutType,
-			}
-			verifyLogContents(podConfig, logPath, expectedLogMessage)
-		})
-	})
+	// Context("runtime should support basic operations on container", func() {
```

Comment:
Hint: you don't need to remove those codes, using `ginkgo` flags instead, e.g. `critest --ginkgo-flags=-focus=Streaming`